### ISSUE TITLE
server crash mech user

### ISSUE THREAD

**Issue by scalda** [rwtema/extrautilities_old_issues#1400]
Mech user, 

crash log from server 
 
[log](http://pastebin.com/Me4357SN)

**Comment by DaBenjle** [comment]
Yep same here. You probably stepped on it while it was on entity mode, correct?

**Comment by scalda** [comment]
correct, was trying to milk cows

**Comment by ktole1999** [comment]
is this being used inside a player claimed area?
